saleor/extensions/base_plugin.py

```diff
@@ -200,6 +200,21 @@ def order_created(self, order: "Order", previous_value: Any):
         """
         return NotImplemented
 
+    def invoice_create(self, order: "Order", invoice: "Invoice", previous_value: Any):
+        """Trigger when invoice is being created.
```

Comment:
The description can lead to misunderstanding. We have empty DB Invoice object passed as an argument. imo we should emphasize that the creation process of the invoice should start here.
What do you think?